webui/src/pages/user/user-namespace-extension-list.tsx

```diff
@@ -0,0 +1,58 @@
+/********************************************************************************
+ * Copyright (c) 2020 TypeFox and others
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License v. 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0.
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ ********************************************************************************/
+
+import * as React from 'react';
+import { Theme, createStyles, WithStyles, withStyles, Typography } from '@material-ui/core';
+import { Namespace } from '../../extension-registry-types';
+import { UserNamespaceExtensionListItem } from './user-namespace-extension-list-item';
+import { ExtensionRegistryService } from '../../extension-registry-service';
+import { ErrorResponse } from '../../server-request';
+
+const extensionListStyles = (theme: Theme) => createStyles({
+    container: {
+        padding: theme.spacing(5, 0),
+    },
+    extensions: {
+        display: 'grid',
+        gridTemplateColumns: `repeat(auto-fit, minmax(200px, 1fr))`,
+        gap: `1.5rem`,
+        marginTop: '1rem',
+    }
+});
+
+const UserNamespaceExtensionListComponent = ({ namespaces, service, setError, classes }: UserNamespaceExtensionListComponent.Props) => {
+    const extensions = Object.keys(namespaces[0].extensions).map((key: any) => namespaces[0].extensions[key]);
+
+    return (
+        <section className={classes.container}>
+            <Typography variant="h5">Extensions</Typography>
+            <div className={classes.extensions}>
+                {
+                    extensions.map((url: any, i) => <UserNamespaceExtensionListItem
```

Comment:
I was not sure what can be the type here I tried `string` as `extensions` is an array of strings but I got the following error: 
```ts
Argument of type '(url: string, i: number) => Element' is not assignable to parameter of type '(value: Extension, index: number, array: Extension[]) => Element'.
Types of parameters 'url' and 'value' are incompatible.
Type 'Extension' is not assignable to type 'string'.ts(2345)
```

Reply:
That is because of the wrong type I've mentioned above.
Anyway, like I've said in other comments, it would be better to pass ready initialised Extension objects here which you should initialise here.